image/image.go

```diff
@@ -263,3 +263,13 @@ func NewImgJSON(src []byte) (*Image, error) {
 	}
 	return ret, nil
 }
+
+// Check wheather id is a valid image ID or not
+func ValidateID(id string) error {
```

Comment:
@thaJeztah `ValidateID` is a docker specific routine to check whether the image id is valid or not and cannot be included in `pkg/stringid` because whole pkgs purpose is about reusable code not tied to docker internals